composer.json

```diff
@@ -19,6 +19,7 @@
     "require": {
         "acquia/acquia-cms-starterkit": "^1.0",
         "drupal/acquia_claro": "^1.3.2",
+        "drupal/core": "^9.5",
```

Comment:
@chandan-singh7929 Isn't this should ^9.5 || ^10?
cc: @vishalkhode1

Reply:
@apathak18 please refer my note above

Reply:
@chandan-singh7929 / @apathak18 I've asked Deepak to remove this as this is not required.